internal/pkg/remote/index.go

```diff
@@ -11,6 +14,48 @@ type Index struct {
 	Components []*model.Component
 }
 
+// NewComponentList - return all published and non-deprecated components.
+// Excluded are processors and code patterns.
+// Components are ordered by ID, first are all from "keboola" vendor.
+func (a *StorageApi) NewComponentList() ([]*model.Component, error) {
```

Comment:
Pridana nova metoda, ktora vrati komponenty, ktore su vhodne na to aby sa vytvoril novy config.
Musia sa ignorovat code patterny, procesory, deprecated a not-published komponenty.
Ako prve uvadzame komponenty od `keboola.`, user bude moct v konzole vyhladavat.